python/kserve/kserve/model_server.py

```diff
@@ -200,10 +227,46 @@ def __init__(
         )
         self._grpc_server = None
         self._rest_server = None
+        self.secure_grpc_server = secure_grpc_server
+        self.grpc_ssl_key = server_key
+        self.grpc_ssl_cert = server_cert
+        self.grpc_ssl_ca_cert = ca_cert
         if self.enable_grpc:
-            self._grpc_server = GRPCServer(
-                grpc_port, self.dataplane, self.model_repository_extension
-            )
+            if self.secure_grpc_server:
```

Comment:
Maybe we can move these validations to a util file?
certs_utils?

Reply:
this would be fine.  could put them in the regular utils file (kserve/utils/utils.py) if you would like - let me know.

Reply:
did not realize there's already a creds_utils file, going to put them in there